core/selection/selection.model.d.ts

```diff
@@ -37,8 +37,9 @@ export declare class SelectionModel {
 	 * * `'single'`
 	 * * `'multiple'`
 	 * * `'range'`
+	 * * `'batch'`
 	 */
-	mode: 'single' | 'mutiple' | 'range';
+	mode: 'single' | 'mutiple' | 'range' | 'batch';
 
```

Comment:
There shouldnt be mode batch

Reply:
![image](https://user-images.githubusercontent.com/13670003/38799004-e96c81e6-416b-11e8-981b-0327f4094e3d.png)

Without 'batch' it won't work